manager/client/main.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	rpc_license "github.com/enfabrica/enkit/manager/rpc"
+	"google.golang.org/grpc"
+	"io"
+	"log"
+	"os"
+	"os/exec"
+	"os/user"
+	"strings"
+	"time"
+)
+
+func run(timeout time.Duration, cmd string, args ...string) {
+	channel := make(chan bool)
+	defer close(channel)
+	go func(status chan bool, cmd string, args ...string) {
```

Comment:
ugh. No need for this either! `os/exec` supports timeouts natively, look at `CommandContext`, I found an example here:
https://medium.com/@vCabbage/go-timeout-commands-with-os-exec-commandcontext-ba0c861ed738

(further down the page, look for CommandContext)

Reply:
Huh, I didn't know that the exec library natively supported timeouts. That is interesting.